documentation/book/con-overview-distributed-tracing.adoc

```diff
@@ -0,0 +1,48 @@
+
+.Introduction to OpenTracing and Jaeger
+
+Distributed tracing in {ProductName} is implemented using the open source {OpenTracingHome} and {JaegerHome} projects.
+
+The OpenTracing specification defines APIs that developers can use to instrument applications -- often in microservice architectures -- for distributed tracing. It is independent from the tracing system.
+
+When instrumented, applications generate __traces__ for individual transactions. Traces are composed of __spans__, which define specific units of work.
+
+To simplify the instrumentation of Kafka Producer, Consumer, and Streams API applications, {ProductName} includes the https://github.com/opentracing-contrib/java-kafka-client/blob/master/README.md[OpenTracing Apache Kafka Client Instrumentation^] library. This library is an example of a __tracer__.
+
+NOTE: The OpenTracing project is merging with the OpenCensus project. The new, combined project is named {OpenTelemetryHome}. OpenTelemetry will provide compatibility for applications that are instrumented using the OpenTracing APIs.
+
+Jaeger, a tracing system, is an implementation of the OpenTracing APIs used for monitoring and troubleshooting microservices-based distributed systems. It consists of four main components and provides client libraries for instrumenting applications. You can use the Jaeger user interface to visualize, query, filter, and analyze trace data.
+
+.An example of a query in the Jaeger user interface
+
+image:image_con-overview-distributed-tracing.png[Simple Jaeger query]
+
+== Distributed tracing support in {ProductName}
+
+In {ProductName}, distributed tracing is supported in:
+
+* Kafka Connect (including Kafka Connect with Source2Image support)
+* Mirror Maker 
+
+You enable and configure distributed tracing for these types of components by setting template configuration properties in the relevant custom resource (for example, `KafkaConnect`).
+
+To enable distributed tracing in Kafka Producer, Consumer, and Streams API applications, you can instrument application code using the OpenTracing Apache Kafka Client Instrumentation library.
+When instrumented, these clients generate traces for messages (for example, when producing messages or writing offsets to the log).
+
+Traces are sampled according to a sampling strategy and then visualized in the Jaeger user interface. This trace data is useful for monitoring the performance of your Kafka cluster and debugging issues with target systems and applications.
+
+Before instrumenting a client application, you must first initialize and configure a Jaeger tracer in the Kafka cluster for use by the application, as described in xref:proc-configuring-jaeger-tracer-kafka-clients-{context}[Initializing a Jaeger tracer for Kafka clients].
+
+NOTE: Distributed tracing is not supported for Kafka brokers.
```

Comment:
I wonder if this should be at the assembly level?

Reply:
Agreed, it works better there.